templates/travel/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,164 @@
+<!-- Hero -->
+{{#partials.hero}}
+<div class="travel-hero-bg">
+  {{#images}}
+    <amp-img
+      class="travel-hero-bg-img"
+      src="../../../img/travel/{{src}}.jpg"
+      height="80vmax"
+    ></amp-img>
+  {{/images}}
+  <div class="travel-hero-bg-fade"></div>
+</div>
+
+<section class="relative travel-hero">
+  <div class="travel-hero-content container flex items-center">
+    <div class="travel-hero-content-inner relative px1 md-px2 flex-auto">
+      <h1 class="travel-hero-heading mb2 line-height-1">Adventures<br class="md-hide lg-hide"/> made&nbsp;easy</h1>
+      <div class="travel-hero-subheading xs-hide sm-hide">Find and book activities, tours, and experiences.</div>
+      <!-- Search Form -->
+      <div class="travel-hero-search">
+        <div class="md-hide lg-hide">
+          <label class="travel-input-icon shadow rounded">
```

Comment:
cleanup unused classes

Reply:
Can you take a pass at the entire PR please

Reply:
This PR was a little messy with regard to namespacing ```travel-shadow``` and ```travel-inline-list```. Sorry about that, should be cleaner in future PRs